doc/z-chap13.xml

```diff
@@ -201,7 +201,20 @@
     <#Include Label = "Normalizer">
  
   </Section>
-  
+
+  <!--**********************************************************************-->
+  <!--**********************************************************************-->
+
+  <Section>
+
+    <Heading>
+      The wreath product of a transformation monoid and a permutation group
```

Comment:
I think this should be in the same section as the documentation for the direct product, if that exists.